README.md

```diff
@@ -4,8 +4,7 @@
 >
 > a semiconductor device with two terminals, typically allowing the flow of current in one direction only
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ochrons/diode)
-[![Build Status](https://travis-ci.org/ochrons/diode.svg?branch=master)](https://travis-ci.org/ochrons/diode)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/suzaku-io/diode)
```

Comment:
There is a Travis CI build for the new repo as well at https://travis-ci.org/suzaku-io/diode